gc_tests/miri_test_setup.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e;
+
+PROJECTDIR=$1
+TMPDIR=$2
+
+# Miri deps cause breakages when used with non-miri targets. So we copy gcmalloc
+# sources over to a tempdir to compile with miri flags so we can nuke it
+# afterwards.
+rsync -a --partial --info=progress2 --exclude .git --exclude target --exclude gc_tests $PROJECTDIR/ $TMPDIR
```

Comment:
`rsync`?!

Reply:
This could probably be `cp` now. I used `rsync` while developing because there was a lot of trial-and-error testing and I wanted to speed up transfer of targets. I'm curious, is the "?!" a comment on the overkill or poor platform compatibility. If the latter, I'm afraid there's plenty more where that came from with this approach to getting Miri running.

Reply:
Yes, `cp` is more portable than `rsync`, and given that we copy to a fresh directory each time, I think `cp` is sufficient.